crd/src/bin/generate_crds.rs

```diff
@@ -0,0 +1,8 @@
+use stackable_kafka_crd::KafkaCluster;
+use stackable_operator::crd::CustomResourceExt;
+
+fn main() {
+    let target_file = "deploy/crd/kafkacluster.crd.yaml";
+    KafkaCluster::write_yaml_schema(target_file).unwrap();
```

Comment:
Could you perhaps add some very simple error checking? I don't think we should make it a habit to have binaries potentially panicking, even if it is not indended for end-users..